home.html

```diff
@@ -77,5 +77,152 @@ <h2 class="profession-title">I'm a Software Developer</h2>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <div class="button-group-2">
+            <a href="#" class="btn btn-gray">HTML</a>
+            <a href="#" class="btn btn-gray">Bootstrap</a>
+            <a href="#" class="btn btn-gray">Ruby</a>
+          </div>
+          <a href="#" class="btn btn-solid-long">See Project</a>
+        </div>
+        <div class="work-item">
+          <h3 class="sub-heading">Profesional Art Printing Data</h3>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <div class="button-group-2">
+            <a href="#" class="btn btn-gray">HTML</a>
+            <a href="#" class="btn btn-gray">Bootstrap</a>
+            <a href="#" class="btn btn-gray">Ruby</a>
+          </div>
+          <a href="#" class="btn btn-solid-long">See Project</a>
+        </div>
+        <div class="work-item">
+          <h3 class="sub-heading">Profesional Art Printing Data</h3>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <div class="button-group-2">
+            <a href="#" class="btn btn-gray">HTML</a>
+            <a href="#" class="btn btn-gray">Bootstrap</a>
+            <a href="#" class="btn btn-gray">Ruby</a>
+          </div>
+          <a href="#" class="btn btn-solid-long">See Project</a>
+        </div>
+        <div class="work-item">
+          <h3 class="sub-heading">Profesional Art Printing Data</h3>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <div class="button-group-2">
+            <a href="#" class="btn btn-gray">HTML</a>
+            <a href="#" class="btn btn-gray">Bootstrap</a>
+            <a href="#" class="btn btn-gray">Ruby</a>
+          </div>
+          <a href="#" class="btn btn-solid-long">See Project</a>
+        </div>
+      </div>
+    </section>
+    <section class="about-section">
+      <h1 class="aboutme-heading">About Me</h1>
+      <p class="aboutme-desc">
+        Hello I’m a software developer! I can help you build a product , feature
+        or website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <a href="#" class="btn btn-solid-short">Get My Resume</a>
+      <hr />
+      <div class="about-categories">
+        <ul class="list-1">
+          <li>Languages</li>
+        </ul>
+        <div class="list-items">
+          <h4>JavaScript</h4>
+          <h4>Ruby</h4>
+          <h4>HTML</h4>
+          <h4>CSS</h4>
+        </div>
```

Comment:
I think the list of project technologies (e.g., "JavaScript", "Ruby", "HTML",  and "CSS" ) should be a list `ul` and each of them a `li`. Please add the `ul` tag instead of the `div` tag and replace the `h4` tag with a `li` tag 😊 .